Bottom: 15d584ee23e38f450359c3caea511818f5bf14c0
Top:    123a60d6442394ee440c1abde7600937c1f536cb
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-17 14:03:54 -0700

Refresh of af_ib

---

diff --git a/src/cma.c b/src/cma.c
index 5280620..9286e81 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -185,9 +185,6 @@ static void ucma_set_af_ib_support(void)
 	struct sockaddr_ib sib;
 	int ret;
 
-	/* Just return - do not enable AF_IB support for now */
-	return;
-
 	ret = rdma_create_id(NULL, &id, NULL, RDMA_PS_IB);
 	if (ret)
 		return;
